query-input/src/main/java/de/brandwatch/minianalytics/queryinput/controller/Controller.java

```diff
@@ -0,0 +1,21 @@
+package de.brandwatch.minianalytics.queryinput.controller;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@org.springframework.stereotype.Controller
+public class Controller {
+
+    Logger logger = LoggerFactory.getLogger(Controller.class);
+
+
+    @RequestMapping(value = "/index")
```

Comment:
You should be explicit about the request method you're using, either by adding `method= RequestMethod.GET`, or even better, just use `@GetMapping` instead.

Reply:
Done